CONTRIBUTING.md

```diff
@@ -3,3 +3,5 @@ Welcome! Thanks for your interest in contributing to Chassis. You're **almost**
 You may also want to take a look at our [commit & pull request guide](http://contribute.jquery.org/commits-and-pull-requests/) and [style guides](http://contribute.jquery.org/style-guide/) for instructions on how to maintain your fork and submit your code. Before we can merge any pull request, we'll also need you to sign our [contributor license agreement](http://contribute.jquery.org/cla).
 
 You can find us on [IRC](http://irc.jquery.org), specifically in #css-chassis should you have any questions. If you've never contributed to open source before, we've put together [a short guide with tips, tricks, and ideas on getting started](http://contribute.jquery.org/open-source/).
+
+We also have weekly meetings on Tuesday at 1pm ET on [IRC](http://irc.jquery.org) in #jquery-meeting. If you can't make the meeting, you can find the internal notes for each meeting on [the jQuery Meeting site](https://meetings.jquery.org/category/chassis/).
```

Comment:
I might change this to just 
"Our weekly meetings are ..." i would also add a note that these are public meetings and all are welcome to attend and participate